src/include/sof/lib/dma.h

```diff
@@ -58,6 +59,7 @@ struct comp_buffer;
 #define DMA_DEV_ALH		BIT(5) /**< connectable to ALH link */
 #define DMA_DEV_SAI		BIT(6) /**< connectable to SAI fifo */
 #define DMA_DEV_ESAI		BIT(7) /**< connectable to ESAI fifo */
+#define DMA_DEV_BUFFER		BIT(8) /**< connectable to buffer component */
 
```

Comment:
This looks misaligned. Maybe, github interface is at fault. But please check on your side.

Reply:
@dbaluta I've double-checked, it's properly aligned in the code. It's just a GH quirk.